src/app/public/modules/footer/footer.component.spec.ts

```diff
@@ -123,7 +123,6 @@ describe('StacheFooterComponent', () => {
     component.ngOnInit();
     fixture.detectChanges();
 
-    expect(component.footerLinks).not.toEqual(footerConfig.nav);
```

Comment:
☝️ Not part of the original scope, but this line was causing the build to fail because the two objects weren't of the same type. Naturally, this line always passed. Also, on the very next line we expect the same object `footerLinks` to equal a different value. Seemed like an unnecessary line for me so I removed it. Let me know if you feel otherwise.